src/main/osd/osd_warnings.c

```diff
@@ -132,10 +134,16 @@ void renderOsdWarning(char *warningText, bool *blinking, uint8_t *displayAttr)
     }
 
     // Warn when in flip over after crash mode
-    if (osdWarnGetState(OSD_WARNING_CRASH_FLIP) && isFlipOverAfterCrashActive()) {
-        tfp_sprintf(warningText, "CRASH FLIP");
-        *displayAttr = DISPLAYPORT_ATTR_INFO;
-        return;
+    if (osdWarnGetState(OSD_WARNING_CRASH_FLIP) && IS_RC_MODE_ACTIVE(BOXFLIPOVERAFTERCRASH)) {
+        if (isFlipOverAfterCrashActive()) { // if was armed in crash flip mode
+            tfp_sprintf(warningText, CRASH_FLIP_WARNING);
+            *displayAttr = DISPLAYPORT_ATTR_INFO;
+            return;
+        } else if (!ARMING_FLAG(ARMED)) { // if disarmed, but crash flip mode is activated
+            tfp_sprintf(warningText, "CRASH FLIP SWITCH");
```

Comment:
Is there a reason why we need a longer message here? We could just use `CRASH_FLIP_WARNING` here as well.

Reply:
When the quad is armed and in "crash flip" it is pretty dangerous. Motors are not spinning, but they will spin if you move sticks a little.
I was even thinking to use something like ">>> CRASH FLIP <<<" to highlight that the drone is ARMED and dangerous.

The idea so that the pilot was able to distinguish between CRASH FLIP ARMED and CRASH FLIP DISARMED.

Reply:
But since the "ARMED" warning is also there when armed, we don't need the extra " SWITCH" characters in the crash flip message when disarmed, imo.

Reply:
Trying to cover all safety scenarios:
The "ARMED" warning appears only for a second. A pilot might flick it accidentally when the quad is in turtle mode without noticing. And then, for example, he sits down, put goggles on, and tries to open "CMS MENU" with stick commands lol.

Reply:
But if "CRASH FLIP" flashes up, no matter if armed of disarmed, you will be reminded to flip it off if you were not intending to use it so it seems unnecessary to me to alter the warning message.

I would rather propose to have "ARMED" flash up constantly if you are in CRASH FLIP mode, as a clean solution.

Reply:
That would be cool to have ARMED flashing. But we do not have "ARMED" OSD elements. It's just a hardcoded behavior to hide everything and show "ARMED" for a half second.

I changes "CRASH FLIP" to "> CRASH FLIP <". It's pretty visible now that something is going on.

Reply:
I see... forget what I said then. I thought it was simpler.

Reply:
We have the DISARMED element:

![image](https://user-images.githubusercontent.com/8344830/170781414-5ba56609-b8aa-415c-9c3b-9f04779c55e5.png)

Reply:
Yeah that still separate element if user wants sine extra info on OSD